clusterview.py

```diff
@@ -351,181 +323,449 @@ def download_file_handling_setup(self):
         else:
             # Handle an actual file
             name = os.path.basename(filename)
             target_file = os.path.join(target_directory, name)
             try:
                 if not os.path.isfile(target_file):
                     # No file name conflict, just move
-                    shutil.move(filename, target_directory)
+                    shutil.move( filename, target_directory )
                     if filename.endswith('.csv'):
-                        # File is .csv, we need to remember
-                        # this one has been handled already
+                        # File is .csv, we need to remember this one has been handled already
                         self.csv_dict[name] = name
                 elif name.endswith('.csv'):
-                    # File exists and is csv.
-                    # Ask the user whether to append or to create a new file
+                    # File exists and is csv. Ask the user whether to append or to create a new file
                     if name not in self.csv_dict:
                         append = self.ask_csv_append(name, has_been_downloaded)
                         if append:
                             self.csv_dict[name] = name
-                            self.handle_csv(filename,
-                                            os.path.join(target_directory,
-                                                         name))
+                            self.handle_csv( filename, os.path.join(target_directory, name) )
                         else:
                             self.csv_dict[name] = self.rename_file(name)
-                            shutil.move(filename, os.path.join(target_directory,
-                                                               self.csv_dict[
-                                                                   name]))
+                            shutil.move( filename, os.path.join(target_directory, self.csv_dict[name]))
                     else:
-                        self.handle_csv(filename, os.path.join(target_directory,
-                                                               self.csv_dict[
-                                                                   name]))
+                        self.handle_csv( filename, os.path.join(target_directory, self.csv_dict[name]))
                 else:
                     # File exists, use a new name
                     new_name = self.rename_file(name)
-                    shutil.move(filename,
-                                os.path.join(target_directory, new_name))
+                    shutil.move( filename, os.path.join(target_directory, new_name))
             except Exception as e:
                 print(e)
                 wx.MessageBox(
                     "Failed to move a file to the destination",
                     caption="File error",
                     style=wx.OK | wx.ICON_INFORMATION)
-                raise e
+                raise(e)
```

Comment:
`raise` is a statement, not a function, so the thing to be raised does not need to be (and should not be) put in parentheses.